pkg/capabilities/cli/cmd/generate_user_types.go

```diff
@@ -13,16 +13,17 @@ func GenerateUserTypes(info UserGenerationInfo) error {
 		return err
 	}
 
+	generatedInfo := GeneratedInfo{}
 	var errs []error
-	for _, file := range dir {
+	for i, file := range dir {
 		fileName := file.Name()
 		if file.IsDir() || !strings.HasSuffix(fileName, ".go") {
 			continue
 		}
 
 		rawContent, err2 := os.ReadFile(path.Join(info.Dir, fileName))
 		if err2 != nil {
-			return err2
+			errs = append(errs, err2)
```

Comment:
nit/ you can build the error inline rather than two steps
var error
if err2 != nil {
err= errors.Join(err, err2)
}
...
if err !=nil {
return err
}

Reply:
Did it for now, lmk if I should change it back.